templates/bookmarks/bookmark_category_attribution.txt

```diff
@@ -0,0 +1,21 @@
+{% load absurl %}Bookmark Category downloaded from Freesound
```

Comment:
I was thinking if it would make sense to reuse some template/method code between packs and bookmark categories. Did you think  about that?

Reply:
I am reusing the pack_attributions.txt file for bookmark categories as well, shall I modify the "pack" naming of the variable in the txt file to find something suitable for both kind of classes? I have made it work to return a proper .txt file in either case using "pack" naming but I don't know if it might be confusing in the future:

![image](https://github.com/user-attachments/assets/e3bff9a7-ab04-4bd3-9d68-aee34afc31ba)

Reply:
I did not realize you were reusing the same file. Then I might rename the file to `multiple_sounds_download_attribution.txt`, and pass `pack` or `bookmark_category` as needed, then check `type` to see if you should use `pack.name` or `bookmark_category.name`.

Reply:
I was not reusing the same file in fact, I just tried after you suggested it xD